src/lib/types.ts

```diff
@@ -36,3 +36,11 @@ export type TextTransformer = (text: string) => string;
  * @description Type Map for All possible output of an Input Filed Validation
  */
 export type ValidationState = 'Valid' | 'InProgress' | 'Invalid';
+
+/**
+ * @description Type for animation configuration
+ */
+export type FlyAnimationConfig = {
+  in: { x: number; y: number; duration: number };
```

Comment:
This { x: number; y: number; duration: number } can a type and can be used with in & out.
Lets check if svelte transition exposes this type, if yes, lets use it.